app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,42 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
+    @user = current_user
+    @post = Post.first
+    respond_to do |format|
+      format.html { render :new, locals: { like: @like } }
+    end
+  end
+
+  def create
+    # new object from params
+    @like = Like.new(post_params)
+
+    # respond_to block
+    respond_to do |format|
+      format.html do
+        if @like.save
+          # success message
+          post = Post.find(post_params[:author_id])
+          Like.update_post_likes_count(post)
+          flash[:success] = 'Liked'
+          # redirect to index
+          redirect_to user_posts_url
+        else
+          flash.now[:error] = 'Error: Like could not be added'
+          # render new
+          render :new, locals: { like: @like }
+        end
+      end
+    end
+  end
```

Comment:
- [ ] As the previous reviewers requested, you should allow users add likes to posts, currently your app does not count likes on a post as expected, the number of likes should be counted on a particular post e.g likes on Facebook or Instagram posts  you see numbers of likes on a particular post

Reply:
@prolajumokeoni , I won't just have enough time. Please

Reply:
Hi, @jssol, I know it can be difficult, and it must have been a busy week. However, reviewers have guidelines to follow and if we do not follow them, it means we have not professionally executed the job. I claimed your pull request again but I will invalidate it because you have not implemented the changes we suggested. I am also invalidating the project because I can see this is your last trial to get this project over the line and I wouldn't want to waste it since you have not made any changes. Kindly make the changes and submit the pull request for another review. 👍